pino.js

```diff
@@ -272,6 +272,18 @@ Object.defineProperty(pinoPrototype, 'isLevelEnabled', {
   value: isLevelEnabled
 })
 
+function getPrettyStream (opts, prettifier) {
+  if (prettifier && typeof prettifier === 'function') {
+    return prettifier(opts).asMetaWrapper(process.stdout)
+  }
+  try {
+    var prettyFactory = require('pino-pretty')
+    return prettyFactory(opts).asMetaWrapper(process.stdout)
```

Comment:
Can you move `asMetaWrapper` in this repository? I don't think it would be needed at all in `pino-pretty` itself.

Reply:
I intend to when I get back to this later this week (*maybe* sometime today).